python/tvm/relay/qnn/op/qnn.py

```diff
@@ -256,7 +256,7 @@ def simulated_quantize(data, output_scale, output_zero_point, axis=-1, out_dtype
     return _make.simulated_quantize(data, out_dtype, output_scale, output_zero_point, axis)
 
 
-def dequantize(data, input_scale, input_zero_point, axis=-1):
+def dequantize(data, input_scale, input_zero_point, axis=-1, out_dtype="float32"):
     r"""Dequantize op
     This operator takes quantized int8 and unit8 as input and produces
```

Comment:
Looks like this description is not correct anymore after all changes in input/output dtypes.

Reply:
Done.